adapters/socketio_berkeley.c

```diff
@@ -650,7 +652,11 @@ int socketio_setoption(CONCRETE_IO_HANDLE socket_io, const char* optionName, con
         }
         else if (strcmp(optionName, "tcp_keepalive_time") == 0)
         {
+#ifdef __APPLE__
```

Comment:
UPDATE: I confirmed these keep-alive options work in OSX (El Capitan), but haven't tried `IPPROTO_TCP, TCP_KEEPALIVE` in other platforms yet